src/main/scala/IOService/Http4sTestServer.scala

```diff
@@ -0,0 +1,109 @@
+package IOService
+import cats._
+import cats.effect._
+import cats.implicits._
+import org.http4s.circe._
+import org.http4s._
+import io.circe.generic.auto._
+import io.circe.syntax._
+import org.http4s.dsl._
+import org.http4s.implicits._
+import org.http4s.server.blaze.BlazeServerBuilder
+import cats.Monad
+import cats.effect.unsafe.implicits.global
+import cats.effect.{Concurrent, ExitCode, IO, IOApp}
+import org.http4s.server.Router
+import org.scalacheck.Gen
+import poker.Deck.startingDeck
+import poker.{Card, Street}
+import poker.Street._
+
+import scala.collection.mutable
+
+object Http4sTestServer extends IOApp {
+
+  /*
+  - Get all the players and cards of current Street - the board state
+  - Post - deal the hole cards -  preflop state
+   */
+
+  // Request -> F[Option[Response]]
+  // HttpRoutes[F] ^^^
+
+  // Get /flop
+  // GET /turn
+  // GET /river
+  // GET /players
+
+  // ??? Need a DB/state
+
+  def genPreFlopBoard(numPlayers: Int): Gen[Preflop] =
+    startingDeck.dealHoleCards(numPlayers).unsafeRunSync()
+
+  def boardState: Flop       = dealFlop(genPreFlopBoard(2).sample.get)
+  def boardStateErr: Preflop = genPreFlopBoard(2).sample.get
```

Comment:
Is `.get` safe here?

Reply:
No. (i mean it can't fail in that instance.)  This whole block of code will be deleted.  I just wanted some data for an ad hoc super ghetto smoke test.  But point taken, I need to get out of this habit.

Reply:
It's all cool. But, it will be a sign of progress in familiarity with referentially transparency when you do not consider it easier to do think referentially opaque. That's the only reason I'm nitpicking here---just pushing a little.